server/src/typeDefs/mutation.ts

```diff
@@ -2,11 +2,14 @@ import { gql } from 'apollo-server'
 
 export const mutation = gql`
   type Mutation {
-    signupUser(name: String!, email: String!, password: String!): AuthPayload!
-    updateUser(data: UserUpdateInput!, userId: String!): User!
-    loginUser(email: String!, password: String!): AuthPayload!
-    forgetPassword(email: String!): Boolean!
-    resetPassword(password: String!, resetPasswordToken: String!): AuthPayload!
-    deleteUser(userId: String!): User!
+    signupUser(email: String!, password: String!): AuthPayload!
+    # id is just for now. In future need get id from session
+    updateUserSettings(userId: Int!, theme: String): UserSetting!
+    addSourceToFeed(feedId: Int!, sourceId: Int!): addSourceFeedRelation!
```

Comment:
https://stackoverflow.com/questions/66368895/how-to-authenticate-through-graphiql-playground